src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketRepository.java

```diff
@@ -17,21 +20,23 @@
     private final BitbucketProject project;
     private final String slug;
     private final RepositoryState state;
-    private List<BitbucketNamedLink> cloneUrls;
+    private List<BitbucketNamedLink> cloneUrls = new ArrayList<>();
     private String selfLink;
 
     @JsonCreator
     public BitbucketRepository(
             @JsonProperty("name") String name,
-            @JsonProperty("links") Map<String, List<BitbucketNamedLink>> links,
+            @CheckForNull @JsonProperty("links") Map<String, List<BitbucketNamedLink>> links,
```

Comment:
What does checkForNull means here?

Reply:
The same thing it means everywhere else in the repo. That consumers of the variable need to handle the case it is null because they can't know if it will be present or not